memecity.game/Game/Components/ItemComponent.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _ITEMCOMPONENT_H
+#define  _ITEMCOMPONENT_H
+#include <ECS.h>
+#include <iostream>
+#include <Engine\MultimediaManager.h>
+
+struct ItemComponent : public memecity::engine::ecs::Component
+{
+	std::string name, description;
+
+	int damage, defence;
```

Comment:
Zou je een Item niet een `StatsComponent` geven? dan gebruiken we overal dezelfde soort stats namelijk.